static/js/search_suggestion.js

```diff
@@ -608,10 +608,10 @@ exports.get_suggestions = function (query) {
 
     var persons = people.get_all_persons();
 
-    suggestions = get_person_suggestions(persons, last, base_operators, 'pm-with');
+    suggestions = get_person_suggestions(persons, last, base_operators, 'sender');
     attach_suggestions(result, base, suggestions);
 
-    suggestions = get_person_suggestions(persons, last, base_operators, 'sender');
+    suggestions = get_person_suggestions(persons, last, base_operators, 'pm-with');
     attach_suggestions(result, base, suggestions);
 
     suggestions = get_person_suggestions(persons, last, base_operators, 'from');
```

Comment:
Structurally, we want to remove below `from:` as well, right, since that's just another spelling for "sender:"?

Reply:
Yes, as for the case of not suggesting `from` queries below when we are suggesting `sender` queries, this has been taken by the following line of code [here](https://github.com/shubham-padia/zulip/blob/94e6743f787864e14c27063c561a9094af76c301/static/js/search_suggestion.js#L197).
```python
    // Be especially strict about the less common "from" operator.
    if (autocomplete_operator === 'from' && last.operator !== 'from') {
        return [];
    }
```
The above ensures that `from` suggestions are shown only when `from` has been decide as an operator. i.e query starts with `from:`. But removing the from` suggestions from below would mean that we will not show a `from` suggestion even when the user types `from:` in the query bar which we don't want I think.